### ISSUE TITLE
Admin interface to review reported violations

### ISSUE THREAD

**Issue by crookedneighbor** [Commander-Spellbook/website-v2#202]
Depends on #200 and #201

Acceptance Criteria
- [ ] api to fetch violations
- [ ] ui to display violations
- [ ] api to update violation status

**Comment by crookedneighbor** [comment]
No longer relevant as the backend system is happening in a separate app now.
